libnavui-maneuver/src/main/res/drawable/mapbox_ic_lane_left_right_left_only.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated by PaintCode - http://www.paintcodeapp.com -->
+<!-- lane right not left -->
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:name="laneRightNotLeft"
+    android:width="30dp"
+    android:height="30dp"
+    android:viewportWidth="30"
+    android:viewportHeight="30">
+    <!-- Group -->
+    <group
+        android:name="group"
+        android:translateX="20.98"
+        android:translateY="15.02">
+
+        <!-- Bezier -->
+        <path
+            android:name="bezier"
+            android:pathData="M -0.03,-11.01 C 0.04,-11.01 0.15,-10.95 0.22,-10.85 C 0.35,-10.72 5.87,-5 5.87,-5 C 5.87,-5 0.3,0.71 0.17,0.84 C 0.1,0.94 -0.01,1 -0.13,1 C -0.36,1 -0.54,0.77 -0.54,0.49 C -0.54,0.42 -0.53,0.36 -0.51,0.3 C -0.48,0.18 0.25,-2.32 0.28,-2.43 C 0.28,-2.81 0.09,-3.01 -0.14,-3.01 C -0.19,-3.01 -0.25,-3.01 -0.29,-3.01 C -0.36,-3.01 -1.3,-3.01 -1.3,-3.01 L -1.25,-7 C -1.25,-7 -0.32,-7 -0.25,-7 C -0.2,-7 -0.14,-7.01 -0.09,-7.01 C 0.14,-7.01 0.33,-7.2 0.33,-7.49 C 0.3,-7.69 -0.43,-10.19 -0.46,-10.3 C -0.48,-10.37 -0.49,-10.43 -0.49,-10.5 C -0.49,-10.78 -0.31,-11.01 -0.08,-11.02 L -0.03,-11.01 Z"
+            android:fillColor="?attr/maneuverTurnIconShadowColor" />
+
+        <!-- Bezier 2 -->
+        <path
+            android:name="bezier2"
+            android:pathData="M -5.98,11.98 L -5.93,-1.45 C -5.93,-1.45 -5.71,-4.98 -2.84,-4.98 L 2.82,-4.98 "
+            android:strokeColor="?attr/maneuverTurnIconShadowColor"
```

Comment:
Apparently support for variables inside groups was only added in API level 24. As a _temporary_ workaround, mapbox/navigation-ui-resources@ed75e5d893626c80d30f996341b1b938315f926b adds a version of this asset and the slight left/slight right one that uses only two paths and no groups. We’ll need to have a larger conversation about maintaining these simplified assets going forward, because they do result in some overhead for anyone who might need to customize the assets or add new lane configurations.

/ref https://github.com/mapbox/navigation-ui-resources/pull/26#discussion_r707558034